panel/__init__.py

```diff
@@ -68,6 +68,14 @@
 from .template import Template  # noqa
 from .widgets import indicators, widget  # noqa
 
+
+def __getattr__(name):
+    if name == "reactive":
+        from .util.warnings import deprecated
+        deprecated("1.4", "panel.reactive", "panel.react")
+        return react
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+
```

Comment:
Just thinking out loud here, but if the intention is to help people who have done `import panel as pn ; pn.reactive.ReactiveHTML(...)` once ReactiveHTML is on pn.react and pn.reactive is a callable rather than a module, can't we instead set the `ReactiveHTML` attribute on the `pn.reactive` callable to `pn.react.ReactiveHTML` (etc.)? (Or to a proxy that warns and then returns ReactiveHTML?)